agent/cmd/root.go

```diff
@@ -0,0 +1,40 @@
+package cmd
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/spf13/cobra"
+)
+
+var (
+	cliExitInterceptor func(code int)
+)
+
+var rootCmd = cobra.Command{
+	Use: "agent",
+}
```

Comment:
In the `tracetest cli` we are going to import this and add as a child of the `tracetest` command. This way you can call it as `tracetest agent start`